docs/contributing/testing.md

```diff
@@ -158,7 +175,7 @@ Instructions for integrating eslint into VSCode can be found [in the eslint-plug
 - Edit your VSCode settings.json:
 
 ```json
-{ 
+{
   //...
```

Comment:
Should we ditch these comments since as was noted they're not valid (i.e. a copy and paste of this wouldn't work)

Reply:
I kind of enjoy the ones beneath this for describing line-by-line what's going on, and I'm struggling to think of a good alternative for this purpose.

However, I'm fine with getting rid of the `//...` indicators of existing code and just changing the description above it to be more clear that this is code that can be added within existing code.

Reply:
I'll think on this and leave making an action for a release at a later time.